website/templates/project/modal_remove_contributor.mako

```diff
@@ -0,0 +1,122 @@
+<link rel="stylesheet" href='/static/css/pages/remove-contributor-page.css'>
+<div id="removeContributor" class="modal fade">
+    <div data-bind="css: modalSize">
+        <div class="modal-content">
+            <div class="modal-header">
+                <button type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span></button>
+                <h3 class="modal-title" data-bind="text:pageTitle"></h3>
+            </div>
+            <div class="modal-body" >
+                <div data-bind="if: canRemoveNode() && !pageChanged()">
```

Comment:
Over in `contributors.mako` I [see](https://github.com/CenterForOpenScience/osf.io/blob/6cc8cb44cc72778227e3b1f38f19353f7b51cd32/website/templates/project/contributors.mako#L56-L60) we're often (but not always) using Knockout's comment control flow, which seems kinda nice to me as it breaks up the code visually, makes it easier to scan. Did you intentionally choose not to use that style here, @billyhunt?